src/test/java/net/sf/jabref/logic/citationstyle/CitationStyleGeneratorTest.java

```diff
@@ -43,4 +44,76 @@ public void testMissingCitationStyle() {
         assertEquals(expected, citation);
     }
 
+    @Test
+    public void testAsciiDocFormat() {
+        String expectedCitation = "[1] B. Smith, B. Jones, and J. Williams, ``Title of the test entry,'' __BibTeX Journal__, vol. 34, no. 3, pp. 45–67, Jul. 2016.\n";
+        BibEntry entry = TestEntry.getTestEntry();
+        String style = CitationStyle.getDefault().getSource();
+        CitationStyleOutputFormat format = CitationStyleOutputFormat.ASCII_DOC;
+
+        String actualCitation = CitationStyleGenerator.generateCitation(entry, style, format);
```

Comment:
Is it possible to check the clipboard? The goal of this test case is to test the copying of (multiple) entries, not the generation using the CitationStyleGenerator.

Think, with this code, the step to a clipboard test is not that far. It is OK for me to check a single entry as the functionality is the same...